frontend/config-assistant/src/helpers/pathHelper.ts

```diff
@@ -3,6 +3,11 @@ import type {Path, PathElement} from '@/model/path';
 export function dataAt(path: Path, data: any): any {
   let currentData: any = data;
 
+  console.log('data at ', path, ' with data ', data);
```

Comment:
```suggestion
```

Reply:
It's still in progress. The tree node table is giving errors when removing an element from an array because the node in the table somehow seems to still be there. Tried >1h to fix it but didn't succeed yet. Will continue tomorrow.